src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -180,9 +190,15 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
      */
     handleChange = (nextEditorState: EditorState) => {
         const { onChange } = this.props;
-
+        const { isFocused } = this.state;
         const activeMention = this.getActiveMentionForEditorState(nextEditorState);
 
+        if (activeMention && activeMention.mentionString?.length > 0 && isFocused) {
+            this.setState({
+                announceActiveMention: true,
```

Comment:
It seems like `announceActiveMention` is never set to `false`. 

Also, does it trigger the screen reader to update the message when the number of contacts changes?